samples/ClassroomLabs/Scripts/BringImageToSharedImageGallery/Readme.md

```diff
@@ -0,0 +1,31 @@
+# Introduction
+This script is used to bring a custom image from an Azure virtual machine (VM) to shared image gallery.
```

Comment:
...from an Azure Virtual Machine (VM) or DevTest Labs Virtual Machine to ....

Reply:
I left out DevTest Labs in purpose since this is specifically being documented right now for Az Labs.  There is confusion among Az Labs users when Dev Test labs, so I was avoiding those terms as much as possible (albeit, there are places in the script that mention DTL).